### PR TITLE
Minor correction

### PR DESCRIPTION
**Pull request by joinimran**
The last sentence removed as it was making no sense. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5874

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: devices/hololens/hololens-recovery.md [hunk 1]
```diff
@@ -110,8 +110,8 @@ The Advanced Recovery Companion is a new app in Microsoft Store restore the oper
 >In the event that a HoloLens 2 gets into a state where Advanced Recovery Companion cannot recognize the device, and it does not boot, try forcing the device into Flashing Mode and recovering it with Advanced Recovery Companion:
 
 1.	Connect the HoloLens 2 to a PC with Advanced Recovery Companion installed.
-1.	Press and hold the **Volume Up and Power buttons** until the device reboots.  Release the Power button, but continue to hold the Volume Up button until the third LED is lit. It will the the only lit LED.
-   1.	The device should be visible in **Device Manager** as a **Microsoft HoloLens Recovery** device:
+1.	Press and hold the **Volume Up and Power buttons** until the device reboots.  Release the Power button, but continue to hold the Volume Up button until the third LED is lit.
+1.	The device should be visible in **Device Manager** as a **Microsoft HoloLens Recovery** device:
```
Comment (JohanFreelancer9) [inline devices/hololens/hololens-recovery.md]:
```suggestion
1.	The device should be visible in **Device Manager** as a **Microsoft HoloLens Recovery** device.
```

**Comment by joinimran** [comment]
@mypil Ready to go live

**Comment by mypil** [comment]
@mattzmsft - PR has been copy edited and is ready for final review. This is based on issue #5874

Thank you.

cc: @kenwith

**Pull request merged**